test/test-framework/pkg/controller/memcached/memcached_controller_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2018 The Operator-SDK Authors
```

Comment:
We don't need this file as part of our test manifests for running `test local --up-local`.
This file is just present on the samples operator repo as an example on how to write unit tests with the fake client for a reconciler.